### PR TITLE
[yaml] Add Beam YAML Examples

### PR DESCRIPTION
**Pull request by eder001**
1x Example: MapToFields and Combine using SQL
1x Example: MapToFields using SQL and the SQL type command
1x Example: Using JDBC connection to read (Mysql) and write (Postgre) in a database

### CODE REVIEW

File: sdks/python/apache_beam/yaml/examples/test_read_mysql_write_postgres.yaml [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+  transforms:
+    - type: ReadFromJdbc
+      name: ReadFromMysql
+      config:
+        driver_class_name: 'com.mysql.cj.jdbc.Driver'
+        jdbc_url: 'jdbc:mysql://database_ip:3306/database_name'
+        username: 'username'
+        password: 'password'
+        read_query: 'select * from table;'
+        driver_jars: 'mysql-connector-j-9.0.0.jar'
```
Comment (Polber) [inline sdks/python/apache_beam/yaml/examples/test_read_mysql_write_postgres.yaml:38]: You can use the `ReadFromMySql` transform to be less verbose and not require staging a driver JAR
```suggestion
    - type: ReadFromMySql
      config:
        url: 'jdbc:mysql://database_ip:3306/database_name'
        username: 'username'
        password: 'password'
        query: 'select * from table;'
```

File: sdks/python/apache_beam/yaml/examples/test_read_mysql_write_postgres.yaml [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+    - type: WriteToJdbc
+      name: WriteToJdbc
+      input: ReadFromMysql
+      config:
+        driver_class_name: 'org.postgresql.Driver'
+        jdbc_url: 'jdbc:postgresql://database_ip:5432/database_name'
+        username: 'username'
+        password: 'password'
+        table: 'table_name'
+        driver_jars: 'postgresql-42.7.4.jar'
```
Comment (Polber) [inline sdks/python/apache_beam/yaml/examples/test_read_mysql_write_postgres.yaml:48]: Same here
```suggestion
    - type: WriteToPostgres
      input: ReadFromMysql
      config:
        url: 'jdbc:postgresql://database_ip:5432/database_name'
        username: 'username'
        password: 'password'
        table: 'table_name'
```

File: sdks/python/apache_beam/yaml/examples/test_read_mysql_write_postgres.yaml [hunk 3]
```diff
@@ -0,0 +1,48 @@
...
+# This examples reads from a public file stored on Google Cloud. This
+# requires authenticating with Google Cloud, or setting the file in
+#`ReadFromText` to a local file.
+#
+# To set up Application Default Credentials,
+# see https://cloud.google.com/docs/authentication/external/set-up-adc for more
+# information
+#
+# The following example reads data from a Mysql database
+# then load the results back to a Postgre database
```
Comment (Polber) [inline sdks/python/apache_beam/yaml/examples/test_read_mysql_write_postgres.yaml:27]:
```suggestion
# then loads the results back to a Postgres database
```

File: sdks/python/apache_beam/yaml/examples/test_read_mysql_write_postgres.yaml [hunk 4]
```diff
@@ -0,0 +1,48 @@
...
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# This examples reads from a public file stored on Google Cloud. This
+# requires authenticating with Google Cloud, or setting the file in
+#`ReadFromText` to a local file.
+#
+# To set up Application Default Credentials,
+# see https://cloud.google.com/docs/authentication/external/set-up-adc for more
+# information
```
Comment (Polber) [inline sdks/python/apache_beam/yaml/examples/test_read_mysql_write_postgres.yaml:24]:
```suggestion
```

File: sdks/python/apache_beam/yaml/examples/test_read_mysql_write_postgres.yaml
Comment (Polber) [inline sdks/python/apache_beam/yaml/examples/test_read_mysql_write_postgres.yaml:1]: I would move this example to the `io` subdirectory

**Review by Polber — CHANGES_REQUESTED** [review]
Left some comments, thanks!

File: sdks/python/apache_beam/yaml/examples/test_read_mysql_write_postgres.yaml
Comment (eder001) [reply sdks/python/apache_beam/yaml/examples/test_read_mysql_write_postgres.yaml:1]: @Polber Done!

**Comment by damccorm** [comment]
@Polber @eder001 what are next steps on this PR?

**Comment by eder001** [comment]
No, it is not... please just disregard

**Pull request closed without merging**